gradle/multiNodeTesting.gradle

```diff
@@ -8,6 +8,11 @@ task testMultiNode(type: Exec) {
     environment "BUILD_DIR", project.buildDir
 
     def args = ['bash', './testMultiNode.sh']
+
+    if(project.hasProperty('test_ssl')) {
```

Comment:
@mmalohlava hm but won't the above be consistent with `if (project.hasProperty("jacocoCoverage")) {` in the same file?